pymc3/backends/report.py

```diff
@@ -119,8 +119,12 @@ def _run_convergence_checks(self, trace, model):
             if rv_name in trace.varnames:
                 varnames.append(rv_name)
 
-        self._ess = ess = ess(trace, var_names=varnames)
-        self._rhat = rhat = rhat(trace, var_names=varnames)
+        try:
+            idata = _to_arviz(trace, log_likelihood=False)
+        except TypeError: # log_likelihood arg requires ArviZ >= 0.7.0
```

Comment:
Maybe do this with
```
if az.__version__ >= '0.7.0':
    idata = az.from_pymc3(trace, log_likelihood=False)
else:
    idata = az.from_pymc3(trace)
```
(without the alias is less confusing IMHO)

Reply:
Wouldn't it be just as easy to make arviz greater than 0.7 be an installation requirement?

Allowing arbitrary arviz versions is going to be very confusing going forward because of the tight coupling.

Reply:
Agree with @rpgoldman, the way out is forward.